assets/large_language_models/autoprompt/environments/autoprompt/context/conda_dependencies.yaml

```diff
@@ -20,9 +20,9 @@ dependencies:
   - scikit-learn==0.24.1
   - mltable
   - transformers==4.21.1
-  - torch==1.11
-  - openai
-  - azure-ai-ml
+  - torch~=1.11
```

Comment:
this will bring torch 1.13.1, just to make sure it is compatible with the code you run on it

Reply:
ack - thanks for the pointer